### DISCUSSION TITLE
Jupyter Notebook Error - No module named Sharpy

### DISCUSSION THREAD

**Discussion by askerp** [ImperialCollegeLondon/sharpy#150] [Q&A] [answered]
Hello,

I have installed SHARPy successfully and have run the 'xbeam' test. But when I downloaded the accompanying notebook 'A prismatic cantilever beam under a tip load', it gives an error as shown below in the image. I have activated the sharpy_env on the anaconda navigator before opening Jupyter notebook.

![Sharpy](https://user-images.githubusercontent.com/74264453/117892376-888f1a00-b2b0-11eb-9042-5728b8c7cfe3.png)

![sharpy_env](https://user-images.githubusercontent.com/74264453/117892438-9fce0780-b2b0-11eb-9d02-c1b7660bf581.png)

TIA!

**Comment by ArturoMS13** [comment, marked as answer]
Hi askper,
have you also loaded the sharpy environment variables (`source bin/sharpy_vars.sh`) before running the jupyter notebook?
Best regards

**Reply by askerp** [reply]
Yes, I have done source bin/sharpy_vars.sh, now it shows: No module named 'model_static_cantilever'

![Screenshot (1217)](https://user-images.githubusercontent.com/74264453/117944957-ba7e9b80-b305-11eb-90b9-ed7eb4a03283.png)

![Screenshot (1216)](https://user-images.githubusercontent.com/74264453/117944961-bb173200-b305-11eb-8abd-2d197647fcbb.png)

The above occurs when I activate the conda (python) env and then open the anaconda navigator, if I activate sharpy_env and then try to open anaconda navigator I get the following error:

![Screenshot (1218)](https://user-images.githubusercontent.com/74264453/117945472-442e6900-b306-11eb-84eb-96c8b691ef90.png)

**Reply by ngoiz** [reply]
It cannot find the model definition file. Have you downloaded the corresponding files in `./example_notebooks/cantilever/`? Make sure your local structure replicates that. Maybe just run it directly from the SHARPy directory without downloading `sharpy/docs/source/content/example_notebooks/`

**Reply by askerp** [reply]
Yes, I accessed the python files from the path mentioned and it worked. Thank you!
